doc/uml/Class_Diagram

```diff
@@ -0,0 +1,49 @@
+@startuml
+class Data{
+-Idenitifer
+-Content
+}
+
+class Tag{
+-Identifier
+-Sortable Ordinal
++rename()
+}
+
+class Database{
+-Tags[]
+-Data[]
+-Relationships[]
++createData()
++deleteData()
++createTag()
++deleteTag()
+}
+
+class Relationship{
+-Relationship(Data, Tag)
```

Comment:
If this is a constructor, it should be public: `+Relatonship(data, tag)`. It might be better to assume constructors at this moment and just place fields with something akin to `data_ptr`, `tag_ptr`, which sort of explicitly tells you that this connects two objects that already exist.